### PR TITLE
fixes OpenGrok-2574 revisit maven patch after jflex update

### PR DESCRIPTION
**Pull request by tarzanek**

### CODE REVIEW

File: opengrok-indexer/pom.xml [hunk 1]
```diff
@@ -259,7 +278,7 @@ Portions Copyright (c) 2017-2018, Chris Fraire <cfraire@me.com>.
                         </goals>
                         <configuration>
                           <filesToInclude>
-                              ${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/java/JavaSymbolTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/javascript/JavaScriptSymbolTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/plain/PlainFullTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/plain/PlainSymbolTokenizer.java
+                              ${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/java/JavaSymbolTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/javascript/JavaScriptSymbolTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/json/JsonSymbolTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/plain/PlainFullTokenizer.java,${basedir}/target/generated-sources/jflex/org/opengrok/indexer/analysis/plain/PlainSymbolTokenizer.java
```
Comment (vladak) [inline opengrok-indexer/pom.xml]: does it perhaps accept the `**` notation ?

**Comment by vladak** [comment]
what about the test case ?

File: opengrok-indexer/pom.xml (hunk 1 above)
Comment (tarzanek) [reply opengrok-indexer/pom.xml]: it could, I only added json so far, without more tests I hesitate to add more

**Comment by tarzanek** [comment]
I will open a test case bug/PR soon, this could be merged

**Comment by tarzanek** [comment]
the deal is that there is a test case, but it didn't work for negative test, I will fix it in next PR

File: opengrok-indexer/pom.xml (hunk 1 above)
Comment (vladak) [reply opengrok-indexer/pom.xml]: Fair enough.

**Comment by vladak** [comment]
Done deal, will wait for the promised negative test case :-)

**Pull request merged**